setup/seleniumInstall.js

```diff
@@ -8,15 +8,18 @@ var path = require('path'),
 
 var seleniumInstall = function (version) {
   return function installSelenium(callback) {
-    //check package.json
-    var pkg = require(path.resolve(__dirname, '../package.json'));
-    if (pkg.dependencies['selenium-webdriver'] === version) {
+
+    var seleniumVersion = require('selenium-webdriver/package.json').version;
```

Comment:
Since the installation uses the `--save` option, it will change package.json, and thus the original comparison will skip the installation next time the same version is requested.